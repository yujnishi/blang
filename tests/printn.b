main() {
    auto i;

    i = 2;
    while ( i <= 10 ) {
        printf("%2d: ",i);
        printn(123456789,i);
        i++;
        putchar('*n');
    }
}
