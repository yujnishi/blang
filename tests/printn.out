 2: 111010110111100110100010101
 3: 22121022020212200
 4: 13112330310111
 5: 223101104124
 6: 20130035113
 7: 3026236221
 8: 726746425
 9: 277266780
10: 123456789
